docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
```

Comment:
@despean does this docker compose file work?

Reply:
Can confirm that `docker-compose up` works. However, there's no documentation included anywhere in the PR yet as a reminder that this is how to use the `docker-compose.yml` file

Reply:
https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2004 

Excellent thanks for checking.